lib/util/cache/repository/impl/base.ts

```diff
@@ -35,29 +60,23 @@ export abstract class RepoCacheBase implements RepoCache {
         );
         return;
       }
-      const oldCache = JSON.parse(data);
+      const oldCache = JSON.parse(data) as unknown;
 
       if (isValidRev12(oldCache, this.repository)) {
-        const compressed = Buffer.from(oldCache.payload, 'base64');
-        const uncompressed = await decompress(compressed);
-        const jsonStr = uncompressed.toString('utf8');
-        this.data = JSON.parse(jsonStr);
-        this.oldHash = oldCache.hash;
-        logger.debug('Repository cache is valid');
+        await this.restoreFromRev12(oldCache);
+        logger.debug('Repository cache is restored from 12 revision');
```

Comment:
```suggestion
        logger.debug('Repository cache is restored from revision 12');
```
sounds more natural ?

Reply:
Yes, it is 😊